### ISSUE TITLE
Add magnet material to geometry

### ISSUE THREAD

**Issue by awhitbeck** [LDMXAnalysis/ldmx-sw#252]
For understanding the effect of signal inefficiencies due to electrons hitting the side HCal, we need to first include material from magnet which might cover this wide angle region.  Need to know roughly what the material is and the dimensions.

**Comment by omar-moreno** [comment]
I received a drawing of the magnet used for HPS.  I modified the size of the shims such that the gap is 14" instead of the 8" that HPS uses. A drawing of the resulting magnet is attached.
[ldmx_magnet.pdf](https://github.com/LDMXAnalysis/ldmx-sw/files/1053126/ldmx_magnet.pdf)

**Comment by omar-moreno** [comment]
Screenshot of magnet loaded into GDML.  At some point, this should be rewritten so tessellated volumes aren't used for the iron blocks. 

![image](https://user-images.githubusercontent.com/3708508/26894594-b31ad3ac-4b73-11e7-8be8-0b8207f0b0de.png)

**Comment by omar-moreno** [comment]
For future reference, I used https://www.solveering.com/InStep/instep.aspx to convert from an STL file directly to GDML. I believe InStep also allows you to convert from GDML --> CAD @awhitbeck.

**Comment by omar-moreno** [comment]
Moving away from using tessellated solids to describe the iron blocks seemed to have solved the following issue: 
```
-------- WWWW ------- G4Exception-START -------- WWWW -------
*** G4Exception : GeomNav1002
      issued by : G4Navigator::ComputeStep()
Track stuck or not moving.
          Track stuck, not moving for 10 steps
          in volume -ldmx_magnet_PV- at point (-574.699,177.8,-721.64) (local point (-574.699,177.8,-321.64))
          direction: (-0.0458547,0.658799,-0.75092) (local direction: (-0.0458547,0.658799,-0.75092)).
          Potential geometry or navigation problem !
          Trying pushing it of 1e-07 mm ...Potential overlap in geometry!

*** This is just a warning message. ***
-------- WWWW -------- G4Exception-END --------- WWWW -------
```

**Comment by omar-moreno** [comment]
New magnet geometry using G4 solids to describe the iron blocks
![magnet](https://user-images.githubusercontent.com/3708508/28284181-f4be1360-6ae4-11e7-8e20-dc1266353f24.png)

**Comment by omar-moreno** [comment]
Running 1000 4 GeV electrons on target with the magnet material in:

real	10m13.133s
user	9m48.037s
sys	0m0.418s

**Comment by omar-moreno** [comment]
Without the magnet material, it's about 45 seconds faster:

**Comment by omar-moreno** [comment]
This issue was resolved by PR 44b98c385ec81d649d2ccb589610b98b6dcc2fdc.
